utils/messages.py

```diff
@@ -18,3 +18,20 @@ def embedded_stats(title, **stats):
         embed.add_field(name=name, value=stat, inline=True)
 
     return embed
+
+
+def embedded_matchday_results(matchday, matches):
+    """Build embedded message with matchday results"""
```

Comment:
Looks like the code here is self-documenting well enough that we don't need to expand on the docstring.  Though I would appreciate explanations on the inputs and outputs.